src/OrchardCore.Modules/OrchardCore.ContentLocalization/GraphQL/LocalizationQueryObjectType.cs

```diff
@@ -13,6 +18,27 @@ public LocalizationQueryObjectType(IStringLocalizer<LocalizationQueryObjectType>
 
             Field(x => x.Culture).Description(S["The culture for your content item."]);
             Field(x => x.LocalizationSet).Description(S["The localization set for your content item."]);
+
+            Field<ListGraphType<ContentItemType>, IEnumerable<ContentItem>>()
+                .Name("Localizations")
+                .Description(S["The localizations of the content item."])
+                .Argument<StringGraphType, string>("culture", "the culture of the content item", string.Empty)
+                .Resolve(x =>
+                {
+                    var culture = x.GetArgument<string>("culture");
+
+                    var context = (GraphQLContext)x.UserContext;
+                    var contentLocalizationManager = context.ServiceProvider.GetService<IContentLocalizationManager>();
+
+                    if (!string.IsNullOrWhiteSpace(culture))
```

Comment:
Is this mean that the culture could not be `Invariant`?!!

/cc @Skrypt

Reply:
Added support for Invariant by setting the default value to null and doing a nullcheck instead.